source/MaterialXCore/Util.cpp

```diff
@@ -55,6 +57,11 @@ bool isValidName(const string& name)
     return it == name.end();
 }
 
+bool isValidNamespace(const string& name)
```

Comment:
Not sure why this is being added back. Seems like a strange name for a method. This is to find if it's valid and **not** namespaced ?

Reply:
Change seems to have originated from this PR: https://github.com/autodesk-forks/MaterialX/pull/1191